src/Features/ERDDAP/Platform/Alerts/index.tsx

```diff
@@ -1,10 +1,10 @@
-import * as React from "react"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)